09_tabular.ipynb

```diff
@@ -629,9 +631,9 @@
     "A second piece of preparatory processing is to be sure we can handle strings and missing data. Out of the box, sklearn cannot do either. Instead we will use fastai's class `TabularPandas`, which wraps a Pandas data frame and provides a few conveniences. To populate a `TabularPandas`, we will use two  `TabularProc`s, `Categorify` and `FillMissing`. A `TabularProc` is like a regular `Transform`, except that:\n",
     "\n",
     "- It returns the exact same object that's passed to it, after modifying the object *in-place*, and\n",
-    "- It runs the transform once, when data is first passed in, rather than lazily as the data is access.\n",
+    "- It runs the transform once, when data is first passed in, rather than lazily as the data is accessed.\n",
     "\n",
-    "`Categorify` is a `TabularProc` which replaces a column with a numeric categorical column. `FillMissing` is an `TabularProc` which replaces missing values with the median of the column, and creates a new boolean column that is set to True for any row where the value was missing. These two transforms are needed for nearly every tabular dataset you will use, so it's a good starting point for your data processing."
+    "`Categorify` is a `TabularProc` which replaces a column with a numeric categorical column. `FillMissing` is a `TabularProc` which replaces missing values with the median of the column, and creates a new boolean column that is set to True for any row where the value was missing. These two transforms are needed for nearly every tabular dataset you will use, so it's a good starting point for your data processing."
```

Comment:
I don't know if this comes up anywhere else - but 1) do we always use median? why / why not? 2) an example how is_missing helps / its intuition

Reply:
Basically the answer is - it doesn't matter. With the extra is_missing column, the tree can use it however it wants. I'll try to clarify